HW0/P4/P4.py

```diff
@@ -1,7 +1,18 @@
-import numpy as np
+﻿import numpy as np
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
 
+def predictS(A, sk, a):
+    return (A * sk) + a
+
+def predictSig(A, sigmak, B):
+    return np.linalg.inv((A * sigmak * (A.transpose())) + (B * B.transpose()))
```

Comment:
Small style issue, but might be nice to user underscores in your two-word variable names